chia/wallet/wallet_state_manager.py

```diff
@@ -1602,7 +1603,8 @@ async def add_coin_states(
         fork_height: Optional[uint32],
     ) -> bool:
         try:
-            await self._add_coin_states(list(set(coin_states)), peer, fork_height)
+            unique_coin_states = sort_coin_states(list(set(coin_states)))
```

Comment:
both before and after does this:
`list(set(coin_states))`
which is deduplicating coins. Can you explain how this change makes a difference in deduplication?

Reply:
I can't find the deduplicating code you are referring to, can you add a link?

Reply:
putting items in a `set` deduplicates them. We do that on the line I put this comment on.

Reply:
Oh I see, you need to check the diff with the release/1.8.2. It doesn't have List(set()) thing there.

Reply:
> putting items in a `set` deduplicates them. We do that on the line I put this comment on.

You only look at the second commit, i guess?

Reply:
yes. I clicked on the link in the email and apparently only saw a single commit